portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -62,4 +73,49 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     // Redirect back to the HTML page.
     response.sendRedirect("/index.html");
   }
+
+  /** 
+  * Returns the number of comments shown entered by the user, or -1 if the 
+  * comment was invalid. Min must be greater than -1 and Max must be greater 
+  * than or equal to min 
+  *
+  * @param request the request received from the form that contains user input
+  * @param parameter the name of the input parameter one is retreiving
+  * @param min used to establish the lower bound of the input
+  * @param max used to establish the upper bound of the input
+  * @return the user's input (number) or -1 if it does not follow guidelines
+  */
+  private int getUserNum(HttpServletRequest request, String parameter, int min, int max) {
```

Comment:
This code looks duplicated from the other servlets; can you move it to a common class and call that from each of your Servlets?